src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -177,5 +177,10 @@ public static bool IsMountEnabled(this IEnvironment environment)
             var mountEnabled = environment.GetEnvironmentVariable(MountEnabled);
             return !string.IsNullOrEmpty(mountEnabled) && string.Equals(mountEnabled, "1");
         }
+
+        public static string GetKubernetesApiServerUrl(this IEnvironment environment)
+        {
```

Comment:
Should this return `null` or throw in environments where the variables below are not defined?